src/service/cli/generate/generate.js

```diff
@@ -28,12 +29,12 @@ const getPictureFileName = (number) => {
   return pictureFileName;
 };
 
-const generateOffers = (count) => (
+const generateOffers = (count, titles, categories, sentences) => (
   Array(count).fill({}).map(() => ({
```

Comment:
```suggestion
  new Array(count).fill({}).map(() => ({
```
При вызове конструктора всегда пиши `new`